tests/test_post_gen_project.py

```diff
@@ -0,0 +1,184 @@
+import os
+import shutil
+import subprocess
+
+import pytest
+from conftest import HOME
+
+from hooks.post_gen_project import (  # noqa: E402
+    copy_journal_template_files,
+    get_cookiecutter_dir,
+)
+
+
+@pytest.mark.parametrize(
+    "cookiecutters",
+    [
+        # test get_cookiecutter_dir when "scikit-package-manuscript"
+        #   cookiecutter exists.
+        # C1: with "scikit-package-manuscript" and other cookiecutters,
+        #   expect return the path to "scikit-package-manuscript"
```

Comment:
let's say "the full path" maybe.   Because of `tmp_dir` the full path cannot be easily tested, but don't worry too much on that.  Just test the non-tmp part of it maybe?